src/plugins/ui_actions/public/types.ts

```diff
@@ -65,7 +65,7 @@ export interface TriggerContextMapping {
   [VISUALIZE_GEO_FIELD_TRIGGER]: VisualizeFieldContext;
 }
 
-const DEFAULT_ACTION = '';
+export const DEFAULT_ACTION = '';
```

Comment:
No issue with this export, just curious why it was needed since you arent using it anywhere in the PR.

Reply:
Removed as it's no longer needed—when i first started working with the actions I did not know how to add in types for new actions, but I figured that out since 👍